src/simplay-web/src/event/ContainerSetLevelEvent.ts

```diff
@@ -12,6 +17,14 @@ export class ContainerSetLevelEvent extends Event {
     super(forId, timestamp, EventAction.CONTAINER_SET_LEVEL, args);
   }
   execute(context: SimplayContext) {
-    throw new Error('Method not implemented.');
+    const entityDisplayObject = getEntityDisplayObjectById(
+      context,
+      this.forId
+    ) as ExtendedDisplayEntity;
+    const entity = context.simulationData.entities.find(
+      (entity) => entity.id === this.forId
+    ) as ContainerEntity;
+    entity.level = this.args.level;
+    entityDisplayObject.informationText.text = `${entity.level} / ${entity.capacity}`;
```

Comment:
Could it be the case that the set level event should also change the visual as it was done in the prototype or is this done with another event?